benchmark/sparse/linear_regression/inference_benchmark.md

```diff
@@ -0,0 +1,82 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```                                                   
+
+### Results
+### Inference Benchmark
+#### Input samples are sparse feature vectors with 10,000 dimensions
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.4                                                      |
+| TensorFlow       | v1.11.0                                                     |
+| MXNet-mkl         | v1.3.0   
+
+#### CPU
+##### Speed
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet Time/Batch) | Keras-TensorFlow Time/Batch)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 64.2 sec | 64.3 sec
+| C5.8XLarge |   0  | 128 | 32.1 sec | 31.4 sec 
+| C5.8XLarge |   0  | 256 | 16.5 sec | 16.3 sec
+| C5.8XLarge |   0  | 512 | 8.3 sec | 8.8 sec 
+| C5.8XLarge |   0  | 1024 | 4.3 sec | 4.4 sec
+
+#### Memory consumed
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (MB) | Keras-TensorFlow (MB)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 1630.8 | 1573.8 |
+| C5.8XLarge |   0  | 128 | 1574.7 | 1561.2 | 
+| C5.8XLarge |   0  | 256 | 1477.8 | 1501.4  |
+| C5.8XLarge |   0  | 512 | 1407.0| 1472.5 |
+| C5.8XLarge |   0  | 1024 | 1336.3 | 1466.8 |
+
+#### GPU
+### Configuration
+##### Input samples are sparse feature vectors with 10,000 dimensions
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.4                                                      |
+| TensorFlow-GPU   | v1.11.0                                                     |
+| MXNet-cu90mkl    | v1.3.0                                                      |
+
+##### Single GPU
+##### Speed
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Batch) | Keras-TensorFlow Time/Batch)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   1  | 64  | 3.0 sec | 2.0 sec
+| P3.8XLarge |   1  | 128 | 2.3 sec | 1.2 sec 
+| P3.8XLarge |   1  | 256 | 1.2 sec | 0.7 sec
+| P3.8XLarge |   1  | 512 | 0.8 sec | 0.5 sec
+| P3.8XLarge |   1  | 1024 | 0.4 sec | 0.4 sec
+
+##### Memory utilization
```

Comment:
Please also remove, this is the CPU memory% when GPU is used for training.

Reply:
Updated header name to indicate GPU utilization